src/main/java/com/gmail/goosius/siegewar/hud/SiegeHUDManager.java

```diff
@@ -6,7 +6,6 @@
 
 import com.gmail.goosius.siegewar.objects.Siege;
 import com.gmail.goosius.siegewar.SiegeWar;
-import com.gmail.goosius.siegewar.hud.SiegeWarHud;
 
 import org.bukkit.Bukkit;
```

Comment:
This import seems to be used in this file, wouldn't this break the HUD?

Reply:
- The code for me shows this as an unused import
- Also when tested the HUD is working ok
- Maybe its because its in the same package?
- Will I add it back just in case or is it ok?